sdcm/cluster.py

```diff
@@ -35,7 +35,7 @@
 from avocado.utils import script
 from datetime import datetime
 from .log import SDCMAdapter
-from .remote import Remote
+from .remote import Remote, RemoteFabric
```

Comment:
This doesn't related with PR subject & patch subject.
It's fine to split the change into another patch in this PR.

RemoteFabric PR  #814 is draft status, you should mention that this PR replied on  #814.
We can't apply this PR before  #814.

Reply:
I agree we should keep those two PRs separated.

Reply:
Some how it is leaked from  PR #814, it is strange. Fixed.